### PR TITLE
reorganizing use cases, other improvements

### PR DESCRIPTION
**Pull request by anaxceron**
Signed-off-by: anaxceron <ana.ceron@broadcom.com>

- reorganized the list of use cases to place them higher up in the file
- improved procedures for clarity and to match content in Zowe  Docs

### CODE REVIEW

**Review by zFernand0 — APPROVED** [review]
Changes LGTM! 😋 
left one small comment 😉

File: packages/zowe-explorer/README.md [hunk 1]
```diff
@@ -15,183 +19,180 @@ The Zowe Explorer extension modernizes the way developers and system administrat
...
+
+### More information
 
 - For the complete Zowe Explorer documentation, see [Zowe Docs](https://docs.zowe.org/stable/user-guide/ze-install.html).
 - Join the **#zowe-explorer** channel on [Slack](https://openmainframeproject.slack.com/) to stay in touch with the Zowe community.
 
 ## Contents
 
-- [Keyboard Shortcuts](#keyboard-shortcuts)
 - [Prerequisites](#prerequisites)
```
[the remark this answers is not in the document]
Comment (anaxceron) [reply packages/zowe-explorer/README.md:44]: excellent point -- done!

**Review by zFernand0 — APPROVED** [review]
thanks for addressing the changes so quickly! 😋

**Review by JamesBauman — DISMISSED** [review]

**Review by zFernand0 — DISMISSED** [review]
make ssense to combine applications and sample use-cases and move the toc further up 😉 
LGTM! 😋

**Review by zFernand0 — APPROVED** [review]

**Review by JillieBeanSim — APPROVED** [review]
LGTM, thanks @anaxceron for the update

**Pull request merged**